Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/bbox.h

```diff
@@ -224,25 +224,22 @@ namespace CGAL {
     }
 
     template<typename PolygonMesh>
-    CGAL::Bbox_3 vertex_bbox_3(typename boost::graph_traits<PolygonMesh>::vertex_descriptor vd,
-                               const PolygonMesh& pmesh)
+    CGAL::Bbox_3 bbox_3(typename boost::graph_traits<PolygonMesh>::vertex_descriptor vd,
+                        const PolygonMesh& pmesh)
     {
-      return vertex_bbox_3(vd, pmesh,
-        CGAL::Polygon_mesh_processing::parameters::all_default());
+      return bbox_3(vd, pmesh, parameters::all_default());
     }
     template<typename PolygonMesh>
-    CGAL::Bbox_3 edge_bbox_3(typename boost::graph_traits<PolygonMesh>::edge_descriptor ed,
-                             const PolygonMesh& pmesh)
+    CGAL::Bbox_3 bbox_3(typename boost::graph_traits<PolygonMesh>::edge_descriptor ed,
+                        const PolygonMesh& pmesh)
     {
-      return edge_bbox_3(ed, pmesh,
-        CGAL::Polygon_mesh_processing::parameters::all_default());
+      return bbox_3(ed, pmesh, parameters::all_default());
     }
     template<typename PolygonMesh>
-    CGAL::Bbox_3 face_bbox_3(typename boost::graph_traits<PolygonMesh>::face_descriptor fd,
-                             const PolygonMesh& pmesh)
+    CGAL::Bbox_3 bbox_3(typename boost::graph_traits<PolygonMesh>::face_descriptor fd,
+                        const PolygonMesh& pmesh)
```

Comment:
I am completely against that sort of abuse of overload. That will lead to compilation errors for any `PolygonMesh` when all three types of descriptors are simple integers.